src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Decompression.cs

```diff
@@ -95,6 +112,15 @@ public static IEnumerable<object[]> DecompressedResponse_MethodNotSpecified_Orig
         public async Task DecompressedResponse_MethodNotSpecified_OriginalContentReturned(
             string encodingName, Func<Stream, Stream> compress, DecompressionMethods methods)
         {
+            _output.WriteLine($"encodingName={encodingName}");
+            _output.WriteLine($"methods={methods}");
+
+            if (!UseSocketsHttpHandler && encodingName == "br")
+            {
+                // Brotli only supported on SocketsHttpHandler.
+                return;
+            }
```

Comment:
Why does this need to be special-cased?  This test is verifying that we get back the original content, so shouldn't that work on handlers that don't support Brotli?

Reply:
You're right. In this case, I don't need to skip it.

Reply:
FYI.  In the case above where you commented...I do need to skip it.  But I removed the check from a different case that didn't need to skip it.

Reply:
Update:

Ubuntu 18.04 is failing now after activating this test.  Looks like it doesn't like "br" in the Content-Encoding fields:

>System.Net.Http.HttpRequestException : An error occurred while sending the request.
---- System.Net.Http.CurlException : Unrecognized or bad HTTP Content or Transfer-Encoding

So, I'm going to disable this test for CurlHandler